CONTRIBUTING.md

```diff
@@ -17,14 +17,14 @@ Matrix chat](https://matrix.to/#/#hpc:ubuntu.com).
 
 ### Table of Contents
 
-* [Using the issue tracker](#using-the-issue-tracker)
-* [Issues and Labels](#issues-and-labels)
-* [Bug Reports](#bug-reports)
-* [Enhancement Proposals](#enhancement-proposals)
-* [Pull Requests](#pull-requests)
-* [Discussions](#discussions)
-* [Documentation Guidelines](#documentation-guidelines)
-* [Licensing](#licensing)
+- [Using the issue tracker](#using-the-issue-tracker)
+- [Issues and Labels](#issues-and-labels)
+- [Bug Reports](#bug-reports)
+- [Enhancement Proposals](#enhancement-proposals)
+- [Pull Requests](#pull-requests)
+- [Discussions](#discussions)
+- [Documentation guidelines](#documentation-guidelines)
+- [Licensing](#licensing)
```

Comment:
`-` instead of `*`? We might want to write down that we prefer dashes somewhere as I know `*` is pretty common to use for bulleted lists.

Reply:
Those still render as bullets as far as I can tell - that was autodone by VScode actually. I'll change them back to asterisks. The autogenerated TOC includes the page title and 'Table of Contents' itself, I removed those and reformatted the indentation.